### DISCUSSION TITLE
Errors on rate-limiting plugin after upgrade to Kong 3.8.0.0

### DISCUSSION THREAD

**Discussion by spirrello** [Kong/kong#13716] [Help]
We just upgraded our Kong instance to 3.8.0.0 and we're now seeing these errors in the logs.  During a load test rather than Kong responding with HTTP 429s its throwing HTTP 500s.

Has anyone else encountered this?

```
"2024/09/27 02:32:25 [error] 2567#0: *63070 [kong] init.lua:427 [rate-limiting] ./rate-limiting/handler.lua:212: arg #3 `value` for function `store_response_header` must be a string or a number, got nil, client: x.x.x.x, server: kong, request: \"POST /som/path HTTP/1.1\", host: \"some.host.com\", request_id: \"asdfasdfdsf\""
```

**Comment by ProBrian** [comment]
Hi @spirrello, could you provide the config of your rate-limiting plugin?

**Reply by spirrello** [reply]
Hi @ProBrian 

I posted an example in this issue and it looks like it was resolved.

https://github.com/Kong/kong/issues/13715

**Reply by ProBrian** [reply]
cool, then I think we could close this.
